os/fs/mount/fs_mount.c

```diff
@@ -148,11 +149,18 @@ static const struct fsmap_t g_bdfsmap[] = {
 #ifdef CONFIG_FS_PROCFS
 extern const struct mountpt_operations procfs_operations;
 #endif
+#ifdef CONFIG_FS_TMPFS
+extern const struct mountpt_operations tmpfs_operations;
+#endif
 
 static const struct fsmap_t g_nonbdfsmap[] = {
 #ifdef CONFIG_FS_PROCFS
 	{"procfs", &procfs_operations},
 #endif
+
+#ifdef CONFIG_FS_TMPFS
+	{"tmpfs", &tmpfs_operations},
```

Comment:
Must be this matched with fs_automount[] for ```"tmpfs"```?
If it is, how about use definition and use it on both?

Reply:
okay